src/libraries/System.Private.CoreLib/src/System/Half.cs

```diff
@@ -420,13 +421,12 @@ public override bool Equals(object? obj)
         /// <summary>
         /// Returns a value that indicates whether this instance is equal to a specified <paramref name="other"/> value.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
AggressiveInlining does improve microbenchmark results, but it introduces code bloat that tends to hurt real workloads. We are careful about using it.

Given that nothing else is annotated with AggressiveInlining in `Half`, I would not introduce one-off here. I think we would need some real world examples to justify it, and likely apply it to number of other methods in this file.

Reply:
Sure. Removed for this PR. Let it be a part of optimizing the whole `Half` type.